### ISSUE TITLE
add voltage range setting for analog

### ISSUE THREAD

**Issue by rupraph** [rupraph/pipo-modules-FW#21]

**Comment by rupraph** [comment]
This is meant to display the right voltage value, depending on the slide button on the board. 

-> The board does not know in which position is the switch on the board (3.3, 5, or 12v)
-> so we need from the UI to save in the config file what is the selected range, and display the right range

**Comment by Leo-Nicolle** [comment]
@rupraph  I dont think it makes sens: when the user changes the voltage on the board, he will forget to change it on the config. Cant we measure it ?

**Comment by rupraph** [comment]
actually, this is only a UI display issue. the board will always see 0-3.3v. But the user might want to see 0-5, 0-12 depedning on the switch. This cannot be measured by the board
Maybe a good tradeoff is to go from 0 to 100% :D

**Comment by Leo-Nicolle** [comment]
I think I dont understand. You want to show the selected voltage on the board for each slot, right ?

**Comment by rupraph** [comment]
Haha :) 
The initial intent was that the slider displays the applied voltage to pipo. depending on the switch position on the board, the user can choose to use a range of 3.3, 5 or 12. The switch actually applies an electrical scaling, bring 5 or 12 down to 3.3 anyway. so the board always see a 3.3 range, but the user migh be using the 12v range.
So the idea was to have a toggle for the slider to display the real applied voltage.

For now, I realize this is really not a prio, an using a 0-100% can be ok.

**Comment by rupraph** [comment]
switched inputs with physical range scaler to % unit
